keysign/QRCode.py

```diff
@@ -317,5 +318,8 @@ def on_released(widget, event):
 if __name__ == '__main__':
     import sys
     logging.basicConfig(level=logging.DEBUG)
-    data = sys.argv[1]
+    if len(sys.argv) > 1:
```

Comment:
While you're at it, can you make it a `try..catch` rather than a length test? Read PEP-020 for the Zen of Python :)

Reply:
That's some wise principles :)
Thank you. :D